common/components/common/event_projects/JoinConferenceButton.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import React from "react";
+import Button from "react-bootstrap/Button";
+import { Glyph, GlyphSizes, GlyphStyles } from "../../utils/glyphs.js";
+import type { Dictionary } from "../../types/Generics.jsx";
+
+type Props = {|
+  buttonConfig: Dictionary<any>,
+  participant_count: number,
+  className: ?string,
+  variant: string,
+|};
+
+/**
+ * Button for presenting conference links with participant count
+ */
+
+class JoinConferenceButton extends React.PureComponent<Props> {
+  constructor(props: Props): void {
+    super(props);
+  }
+
+  render(): React$Node {
+    return (
+      <React.Fragment>
+        <Button
+          variant={this.props.variant}
+          type="button"
+          className={this.props.className}
```

Comment:
Not for this release but let's mark a todo here -- I should move the CSS that handles this button's internal layout (currently `.AboutEvent-livebutton` and `.AboutEvent-usercount`) into a specific JoinConferenceButton class/classes, and we use that here alongside any classes passed from props.

Reply:
I think it'll work best if I take care of this in #870 after this gets merged to master. Make sense to you?